po/messages.pot

```diff
@@ -1227,11 +1400,11 @@ msgid ""
 "possible. Firmware is extremely complex; UEFI is roughly equivalent in size "
 "and capability to a Unix kernel. Firmware is usually closed and proprietary, "
 "with nowhere near the level of testing of kernels. These properties make "
-"firmware an ideal place for so-called advanced persistent "
-"threats[^10][^18][^5]. Once an exploit is installed, it is almost impossible "
-"to remove, since the exploit can inhibit its removal by corrupting the "
-"firmware update process. The only sure way to mitigate a firmware exploit is "
-"to destroy the hardware."
+"firmware an ideal place for so-called advanced persistent threats[^10] [^18] "
+"[^5]. Once an exploit is installed, it is almost impossible to remove, since "
+"the exploit can inhibit its removal by corrupting the firmware update "
+"process. The only sure way to mitigate a firmware exploit is to destroy the "
+"hardware."
```

Comment:
So every time the original English text is modified then the pot files need to be modified?